### DISCUSSION TITLE
Are the trivy repo's down?

### DISCUSSION THREAD

**Discussion by stumueller** [aquasecurity/trivy#7290] [Bugs]
### Description

when doing an apt update

Err:2 https://aquasecurity.github.io/trivy-repo/deb noble/main amd64 trivy amd64 0.54.0
  404  Not Found [IP: 185.199.109.153 443]

### Desired Behavior

perform apt update

### Actual Behavior

Err:2 https://aquasecurity.github.io/trivy-repo/deb noble/main amd64 trivy amd64 0.54.0
  404  Not Found [IP: 185.199.109.153 443]

### Reproduction Steps

```bash
n/a
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
n/a
```

### Operating System

Ubuntu 24.04

### Version

```bash
0.53.0
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by afdesk** [comment]
it looks like `0.54.1` was cut, so you can use this one

**Reply by afdesk** [reply]
i managed to install:
```sh
$ sudo apt update
$ sudo apt install trivy
$ trivy -v
Version: 0.54.1

```

**Reply by stumueller** [reply]
thank you

**Reply by afdesk** [reply]
@stumueller let's close it?
